src/util/loanUtils.js

```diff
@@ -317,3 +317,50 @@ export function loanCallouts(loan, categoryPageName) {
 
 	return callouts;
 }
+
+/**
+ * Enables session cookie
+ *
+ * @param	{String}	cookieName	The cookie name
+ * @param	{Object}	cookieStore The cookie store
+ * @param	{String}	expDate		The expiration
+ * @returns {void}
+ */
+export function enableCookie(cookieName, cookieStore, expDate) {
+	if (!cookieStore.get(cookieName)) {
```

Comment:
We probably don't want this check here, since if the user clicks the email link again with the UTM, we'll want to set a new expiration date to 24 hours in the future.